pkg/fuzzy/google_translate.go

```diff
@@ -113,83 +117,45 @@ func (g *GoogleTranslate) Translate(
 		return nil, nil //nolint:nilnil
 	}
 
-	// TODO: Implement translation of message texts.
-
 	// NOTE: temporary fix to avoid failing tests.
```

Comment:
i guess this is redundnat now ?

Reply:
I think we can keep it, just the comment has to go.